src/base/tz_compile.c

```diff
@@ -6605,37 +6606,50 @@ tzc_update (TZ_DATA * tzd, const char *database_name)
 	      goto exit;
 	    }
 
+	  error = db_query_get_tuple_value (result1, 1, &value2);
+	  if (error != NO_ERROR)
```

Comment:
Isn't it a problem not to clear through db_value_clear() inside the tzc_update() function?

Reply:
There is a lot of code that calls the db_query_get_tuple_value function and does not call the db_value_clear function.

Since the db_query_get_tuple_value function gets data from a cursor, it seems that calling the db_value_clear function is not necessary.
I will check further.

Reply:
If you look at the function _op_get_constrain_info(), it's always clear.

Reply:
Change it to calling a function instead of executing a query.